worker/src/RTC/ActiveSpeakerObserver.cpp

```diff
@@ -308,10 +325,11 @@ namespace RTC
 			dominantSpeaker->EvalActivityScores();
 			double newDominantC2 = C2;
 
-			for (const auto& it : this->mapProducerSpeaker)
+			for (auto& kv : this->mapProducerSpeakers)
```

Comment:
can't it be `const` ?

Reply:
This is for consistency with the rest of the code. If we change this we should change it everywhere in tons of files.